collector/src/main/java/com/bitmovin/analytics/BitmovinAnalyticsConfig.java

```diff
@@ -298,7 +298,8 @@ public void setTitle(String title) {
     }
 
     /**
-     * MPD URL of the manifest. Usually auto-detected, but can be overwritten.
+     * MPD URL of the manifest.
```

Comment:
nit pick: the method is setting the url - your comments reads like a getter ;)

Reply:
Fixed in c6b4cab34ab6a0906c62e470935495470f95aa19